package.json

```diff
@@ -6,12 +6,14 @@
   "homepage": "https://hypothes.is",
   "bugs": "https://github.com/hypothesis/client/issues",
   "repository": "hypothesis/client",
-  "dependencies": {},
+  "dependencies": {
+    "tailwindcss": "^2.2.19"
```

Comment:
This should be added to `devDependencies`.

Reply:
🤦🏻‍♀️